invenio_theme/bundles.py

```diff
@@ -39,12 +39,12 @@
 from invenio_assets import BowerBundle
 
 css = BowerBundle(
-    'less/invenio_theme/styles.less',
-    filters='less, cleancss',
+    'scss/invenio_theme/styles.scss',
+    filters='sass, cleancss',
```

Comment:
Yep, the syntax is scss but the sass filter is in use. The 'scss' filter is the correct filter to be used with scss syntax: http://webassets.readthedocs.org/en/latest/builtin_filters.html#scss